Calibration/HcalCalibAlgos/plugins/HcalIsoTrkAnalyzer.cc

```diff
@@ -593,15 +598,15 @@ void HcalIsoTrkAnalyzer::analyze(edm::Event const& iEvent, edm::EventSetup const
                          trkCaloDets,
                          geo,
                          caloTopology,
-                         theHBHETopology,
+                         theHBHETopology_,
```

Comment:
As `theHBHETopology_` and `respCorrs_` are class members, why do you pass them as arguments to the (private) method `fillTree`?